README_JP5.md

```diff
@@ -118,24 +118,47 @@ scp ./images/5.0.2/drivers/media/v4l2-core/videobuf-vmalloc.ko nvidia@10.0.0.116
 
 Copy them to the right places on Jetson target:
 ```
-sudo cp Image /boot
-sudo cp tegra194-p2888-0001-p2822-0000.dtb /boot/dtb
-sudo cp d4xx.ko /lib/modules/$(uname -r)/kernel/drivers/media/i2c/
-sudo cp uvcvideo.ko /lib/modules/$(uname -r)/kernel/drivers/media/usb/uvc/
-sudo cp videobuf-core.ko /lib/modules/$(uname -r)/kernel/drivers/media/v4l2-core/
-sudo cp videobuf-vmalloc.ko /lib/modules/$(uname -r)/kernel/drivers/media/v4l2-core/
+sudo cp Image /boot/d457/
```

Comment:
should we also add instructions to create below folders if not present:
`/boot/d457`
`/lib/modules/$(uname -r)/updates/`

Reply:
done.